src/pages/index.tsx

```diff
@@ -73,9 +73,9 @@ const MainSection = () => (
   </Stack>
 )
 
-const AboutPage = () => (
+const HomePage = () => (
```

Comment:
I think using the term Homepage is clearer that About. What do you think?

Reply:
Yes, you are right! I'll rename it later.

Reply:
I forgot about this! You flagged it as resolved so I didn't realize this was pending. Let me fix it in a separate PR.

Reply:
Ah damn, it was renamed already 🤦 . I thought it was pending.

Reply:
I have renamed the component for comparator page tho https://github.com/Belco90/octoclairvoyant/commit/d0482b61910e3e615215acf181823891ea5ead3b